### PR TITLE
Cesium externs

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: Cesium.externs.js [hunk 1]
```diff
@@ -1506,14 +1506,57 @@ Cesium.TilingScheme.prototype.rectangle;
...
+
+
+/**
+ * @type {Cesium.Ellipsoid}
+ */
+Cesium.GeographicTilingSchemeOptions.prototype.ellipsoid;
+
+
+/**
+ * @type Cesium.Rectangle{}
```
Comment (elemoine) [inline Cesium.externs.js]: `{}` problem here.

Comment (gberaudo) [inline Cesium.externs.js]: Indeed, it is surprising closure did not catched it.

**Comment by elemoine** [comment]
Please merge.

**Comment by gberaudo** [comment]
Thank you for the review @elemoine.

**Pull request merged**